2020/day3/solution.py

```diff
@@ -0,0 +1,43 @@
+file = open('input.txt')
+
+class grid:
```

Comment:
```suggestion
class Grid:
```
style nit: classes should be CamelCase. (There are sometimes cases to have snake_case classes when you really want it to be look like a function e.g. some context manager implementations but those mostly happen in library code.)